config/sync/workflows.workflow.content_workflow.yml

```diff
@@ -0,0 +1,46 @@
+uuid: 1eacdf5b-5ad8-411a-85cf-b3395557be99
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.article
+    - node.type.page
+    - node.type.recipe
+  module:
+    - content_moderation
+id: content_workflow
+label: 'Content Workflow'
+type: content_moderation
+type_settings:
+  states:
+    draft:
+      label: Draft
```

Comment:
From the looks of this you reused the state of draft, you should remove this and create a new state called unpublished as per the task in notes.

Reply:
Pull request should not have any instances of draft.

Reply:
done